src/style.css

```diff
@@ -2,0 +1,104 @@
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf');
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf');
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  color: #3d4e61;
+  font-family: Avenir, sans-serif;
+}
+
+.big-form {
+  margin: 20px 15px;
+  position: relative;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.big-form:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.big-form:focus {
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.big-input {
+  padding: 25px 0 25px 62px;
+  width: 100%;
+  outline: none;
+}
+
+.big-input:focus {
+  font-weight: 900;
+}
+
+.big-form::before {
+  content: '';
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  bottom: 26px;
+  background: url(images/Search.svg);
+  width: 19px;
+  height: 19px;
+}
+
+.small-form {
+  margin: 20px 15px;
+  position: relative;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.small-form::before {
+  content: '';
+  position: absolute;
+  left: 13px;
+  top: 15px;
+  bottom: 16px;
+  background: url(images/Icon.svg);
+  width: 11px;
+  height: 11px;
+}
+
+.small-form:focus {
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
```

Comment:
Similar to the '.big-form', the ':focus' pseudo-class on '.small-form' might not work as expected unless the div is made focusable. Ensure this is the desired behavior or adjust the selector to target focusable elements within the form.